pagopa/pn-frontend
# Chore/tests and packages

## Short description
This pr includes two fixes:

1. Tests about calendar on mobile device were enabled and fixed
2. Updated prettier to manage the most recently typescript updates

## List of changes proposed in this pull request
- Updated FilterNotifications.test.tsx
- Updated package.json

## How to test

- Run tests on pf/pa/pg and check that everything is ok
- Go to SercqSendCourtesyDialog.tsx (pf/pg) change imports order and save -> check that the previous order is restored and that prettier works well without errors (on VsCode there is the tab prettier on the right bottom corner. click on it and check the logs).
